### ISSUE TITLE
Tide haults auditing when it encounters a plugin not on WP.org

### ISSUE THREAD

**Issue by bbonahoom** [wptide/wptide.org#130]
## Bug Description

 PHP Compatibility Scanner v1.6.2 Scan stuck on Gravity Forms 2.7.9 after casting an error (Response status: 404  Message: The audit requested does not exist.) for Gravity Edit 2.0
## Expected Behaviour

Complete scan and move on to other plugins
## Steps to reproduce

1. Start scan

## Screenshots

![image](https://github.com/wptide/wptide.org/assets/138803194/4d50ee59-e43e-4abe-a53c-a1fb7311696b)

## Additional context

- Tide version:
- Node version:
- Docker version (if applicable):
- OS:
- Browser: [e.g. chrome, safari]
- Device: [e.g. iPhone8]

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Changelog entry

* 

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

*

**Comment by jeffpaul** [comment]
@bbonahoom https://wordpress.org/plugins/search/Gravity%2520Edit/ does not appear to be a valid plugin on the wp.org repo. I'm closing this as Tide only audits plugins available publicly on the wp.org repo.

**Comment by bbonahoom** [comment]
@jeffpaul - you missed the point of my problem.  Sorry if I didn't state it clear enough.  Tide didn't skip over the plugin.  It. Got. Stuck.   My expected behavior is that it call the scan complete and move on.  Not to get stuck on plugins it doesn't recognize.

**Comment by pbiron** [comment]
> My expected behavior is that it call the scan complete and move on. Not to get stuck on plugins it doesn't recognize.

I completely agree with the above!  Not only are there premium plugins (eg. Gravity Forms), but there are also "site-specific" plugins that aren't going to be in the .org repo.

For any plugin encountered during a scan that is not in the .org repo, the [PHP Compatibility Checker](https://wordpress.org/plugins/php-compatibility-checker/) plugin should simply report "Plugin X is not in the .org repo and so we can't report which versions of PHP it is compatible with." and go on to the next plugin.

As is, I can no longer scan **any** site I work on, as they **all** have either premium or "site-specific" plugins (and most have both).
